src/main/pages/setup-docker/docker-config.adoc

```diff
@@ -472,56 +497,68 @@ CHE_HTTPS_PROXY_FOR_WORKSPACES=<YOUR_PROXY_FROM_DOCKER>
 CHE_NO_PROXY_FOR_WORKSPACES=localhost,127.0.0.1,<YOUR_CHE_HOST>
 ----
 
-The last three entries are injected into workspaces created by your users. This gives your users access to the Internet from within their workspaces. You can comment out these entries to disable access. However, if that access is turned off, then the default templates with source code will fail to be created in workspaces as those projects are cloned from GitHub.com. Your workspaces are still functional, we just prevent the template cloning.
+The last three entries are injected into workspaces created by your users. This gives your users access to the Internet from within their workspaces. You can comment out these entries to disable access. However, if that access is turned off, the default templates with source code fails to be created in workspaces as those projects are cloned from GitHub. Your workspaces are still functional and only  template cloning is prevented.
```

Comment:
* don't use 'however'
* templates ... fails

Reply:
@rkratky , when saying "templates ... fails" you meant that it should have been "templates...fail", correct?

Reply:
yes